examples/window.rs

```diff
@@ -2,12 +2,13 @@
 
 use std::error::Error;
 
-use winit::application::ApplicationHandler;
-use winit::event::WindowEvent;
-use winit::event_loop::{ActiveEventLoop, EventLoop};
+use winit::event_loop::EventLoop;
 #[cfg(web_platform)]
 use winit::platform::web::WindowAttributesExtWeb;
-use winit::window::{Window, WindowAttributes, WindowId};
+use winit_core::application::ApplicationHandler;
+use winit_core::event::WindowEvent;
+use winit_core::event_loop::ActiveEventLoop;
+use winit_core::window::{Window, WindowAttributes, WindowId};
```

Comment:
Hmm, I think we'll want to re-export `winit-core` types in `winit`? So the example changes actually shouldn't be necessary?

Reply:
I'll check at the later stage whether we actually need so and how the reexport should be done..

Reply:
I think you should maybe do it in the opposite order: re-export everything for now with maybe a TODO comment, that way this PR can be a lot smaller; and maybe remove some re-exports later.